lib/overload.pm

```diff
@@ -1,8 +1,12 @@
 package overload;
 
-our $VERSION = '1.32';
+use strict;
+no strict 'vars'; # Needed for assignment to typeglobs and symbol table
```

Comment:
This comment is not accurate; assigning to typeglobs and symbol table entries works fine under strict 'vars'.

Reply:
What would you say instead?

Reply:
That depends why it's needed here. Strict 'vars' requires you to declare a (scalar, array, or hash) variable with `my` or `our` before using it.

Reply:
yep. nevermind. fixed.